tests/noliteral/FloatInference.java

```diff
@@ -0,0 +1,10 @@
+import org.checkerframework.checker.noliteral.qual.*;
```

Comment:
I found the profusion of tiny test files confusing.  It's better to organize the tests.  For example, put all the `*Inference` tests in a single file since they are testing essentially the same thing.  Then it's easier to look over the list of tests and understand what they are testing and what they are not.